docs/ADMIN.md

```diff
@@ -56,7 +56,7 @@ and oauth service to have the same value for them):
 |-------------------------------------------------------|--------------------------------|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | --base-url                                            | BASEURL                        |                          | This is the publicly accessible URL on which the SPI OAuth service is reachable. Note that this is not just a hostname, it is a full URL including a scheme, e.g. "https://acme.com/spi"                                           |
 | --config-file                                         | CONFIGFILE                     | /etc/spi/config.yaml     | The location of the configuration file.                                                                                                                                                                                            |
-| --spi-instance-id                                     | SPIINSTANCEID                  | spi-1                    | ID of this SPI instance. Used to avoid conflicts when multiple SPI instances uses shared resources (e.g. secretstorage).                                                                                                           |
+| --instance-id                                         | INSTANCEID                     | spi-1                    | ID of this SPI instance. Used to avoid conflicts when multiple SPI instances uses shared resources (e.g. secretstorage).                                                                                                           |
```

Comment:
Should the description of the argument change as well? Can we call our whole system (service, spi operator, RS operator) spi ? If no, do we have some different naming?

Reply:
Correct me if I am wrong. We are changing the name of this argument from `--spi-instance-id` to `--instance-id ` because now the RS-operator needs its own ID as well as SPI-operator.

The description says `ID of this SPI instance.` What is meant by `SPI instance` is it just SPI-operator or both SPI-operator and RS-operator?

Reply:
Ok, I think I understand now, the problem was in my understanding. We can keep the description how as it is now.